releases/v0.4.0.md

```diff
@@ -1,12 +1,18 @@
 # Release Notes for v0.4.0
 Release Date: 2023-03-03 
 
-Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential Containers
+Please see the [quickstart guide](../quickstart.md) for details on how to try out Confidential
+Containers
+Please refer to our [Acronyms](https://github.com/confidential-containers/documentation/wiki/Acronyms)
```

Comment:
I believe that without an intermediate white line, this will show as a single paragraph. So either insert a dot at the end of the previous sentence, or insert en empty line to make two paragraphs.

Reply:
Thanks. I think I caught that in the last commit https://github.com/confidential-containers/documentation/pull/102/commits/192959138df40f8ed192ec5eefdcf228bc7e69d5

If I'm missing something tho lemme know.

Reply:
Looks good to me.